core/templates/components/ratings/rating-display/rating-display.directive.spec.ts

```diff
@@ -49,6 +50,7 @@ describe('Rating display directive', function() {
       outerScope.$digest();
       ctrlScope = compiledElem[0].getControllerScope();
     }));
+
   it('should display the correct number of stars', function() {
```

Comment:
For this test what are the pre-checks, actions, and post-checks? Can you add a comment in the code mentioning the same?

Reply:
Done